packages/react/package.json

```diff
@@ -10,17 +16,14 @@
     "lib"
   ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: run tests in xing/hops repo\" && exit 1"
   },
-  "keywords": [],
-  "author": "",
-  "license": "MIT",
   "peerDependencies": {
     "hops-config": "6.1.0",
-    "react": "^15.5.4",
+    "react": "^15.6.1",
```

Comment:
Shouldn't we be more conservative with the peerDependencies and e.g. allow react down to 15.4?